python/lsst/afw/table/baseColumnView.cc

```diff
@@ -22,24 +22,89 @@
 
 #include <pybind11/pybind11.h>
 //#include <pybind11/operators.h>
-//#include <pybind11/stl.h>
+#include <pybind11/stl.h>
+
+#include "numpy/arrayobject.h"
+#include "ndarray/pybind11.h"
+#include "ndarray/converter.h"
+
+#include "lsst/afw/table/Source.h"
+#include "lsst/afw/table/Key.h"
+#include "lsst/afw/table/BaseColumnView.h"
+
+namespace ndarray { namespace detail {
+template <> struct NumpyTraits<lsst::afw::geom::Angle> : public NumpyTraits<double> {};
```

Comment:
I had never seen this before either. I learned a lot of new C++ in the process of wrapping this module!